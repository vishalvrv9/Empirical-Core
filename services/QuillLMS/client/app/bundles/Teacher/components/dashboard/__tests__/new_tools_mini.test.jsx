import React from 'react';
import { shallow } from 'enzyme';

import NewToolsMini from '../new_tools_mini';

describe('NewToolsMini component', () => {

  // Update this snapshot whenever we update the NewToolsMini.
  it('should render', () => {
    expect(shallow(<NewToolsMini />)).toMatchSnapshot();
  });

});
