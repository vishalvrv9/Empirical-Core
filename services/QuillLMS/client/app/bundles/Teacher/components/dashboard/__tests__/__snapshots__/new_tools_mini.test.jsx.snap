// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewToolsMini component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "position": "relative",
        }
      }
    >
      <img
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "27px",
            "position": "absolute",
            "right": "-22px",
            "top": "-11px",
            "transform": "rotate(-22deg)",
            "width": "27px",
          }
        }
      />
      <img
        src="https://assets.quill.org/images/icons/icon-star.svg"
        style={
          Object {
            "height": "14px",
            "position": "absolute",
            "right": "-3px",
            "top": "-11px",
            "transform": "rotate(-34deg)",
            "width": "14px",
          }
        }
      />
      <h3>
        Two New Writing Tools
      </h3>
    </div>
    <div>
      <a
        href="/tools/diagnostic"
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
            "marginTop": "8px",
          }
        }
      >
        <img
          src="/images/icon-diagnostic-black.svg"
          style={
            Object {
              "height": "25px",
              "marginRight": "10px",
              "marginTop": "3px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "18px",
              "fontWeight": "bold",
              "margin": "8px 0",
            }
          }
        >
          Quill Diagnostic
        </p>
      </a>
      <a
        href="/tools/connect"
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
          }
        }
      >
        <img
          src="/images/icon-connect-black.svg"
          style={
            Object {
              "height": "26px",
              "marginRight": "10px",
            }
          }
        />
        <p
          style={
            Object {
              "fontSize": "18px",
              "fontWeight": "bold",
              "margin": "8px 0",
            }
          }
        >
          Quill Connect
        </p>
      </a>
    </div>
    <p
      style={
        Object {
          "marginTop": "12px",
          "padding": "0px 15px",
        }
      }
    >
      Use our new tools to assess your studentsâ€™ writing and build sentence structure skills.
    </p>
    <a
      href="/tools/diagnostic"
    >
      <button
        className="button button-white beta"
        style={
          Object {
            "marginTop": "18px",
          }
        }
      >
        Learn More
      </button>
    </a>
  </div>
</div>
`;
